packages/design-system/src/components/loadingSpinner/stories/index.js

```diff
@@ -25,14 +22,18 @@ import { LoadingSpinner } from '..';
 export default {
   title: 'DesignSystem/Components/LoadingSpinner',
   component: LoadingSpinner,
+  args: {
+    animationSize: 95,
+    circleSize: 12,
+    numCircles: 11,
+  },
+  parameters: {
+    controls: {
+      exclude: ['loadingMessage'],
+    },
+  },
 };
 
-export const _default = () => {
-  return (
-    <LoadingSpinner
-      animationSize={number('Animation Size', 95)}
-      circleSize={number('Circle Size', 12)}
-      numCircles={number('Number of Circles', 11)}
-    />
-  );
+export const _default = (args) => {
+  return <LoadingSpinner {...args} />;
```

Comment:
For later: let's add a container around this so it's not cut off at the edges
![image](https://user-images.githubusercontent.com/22185279/151242805-dded089d-86e7-467a-84b4-57990b812b31.png)